src/monitor-plan/monitor-plan.repository.ts

```diff
@@ -51,6 +52,28 @@ export class MonitorPlanRepository extends Repository<MonitorPlan> {
       .getMany();
   }
 
+  async getMonitorPlanUnitStackConfigs(planId: string) {
+    const configIds = (
+      await this.query(
+        `
+      SELECT config_id FROM camdecmps.unit_stack_configuration usc
```

Comment:
As discussed, this logic has been updated to avoid using this particular view, and to instead get a plan's unit stack configurations based on its locations and begin and end dates. This logic has been moved to `getUnitStackConfigsByMonitorPlanId` in `UnitStackConfigurationRepository` and `UnitStackConfigurationWorkspaceRepository`.